src/Header/SetCookie.php

```diff
@@ -507,9 +513,9 @@ public function setHttponly($httponly)
     }
 
     /**
-     * @return bool
+     * @return null|bool
      */
-    public function isHttponly()
+    public function isHttpOnly()
```

Comment:
Same comment here as for `setHttponly()`; revert, or introduce the new method as a proxy to the original.